plugins/modules/zos_tso_command.py

```diff
@@ -175,19 +210,37 @@ def run_module():
         module.fail_json(msg=repr(e), **result)
 
     commands = parsed_args.get("commands")
+    max_rc = parsed_args.get("max_rc")
+    if max_rc is None:
+        max_rc = 0
 
     try:
-        result["output"] = run_tso_command(commands, module)
+        result["output"] = run_tso_command(commands, module, max_rc)
+        result["max_rc"] = max_rc
```

Comment:
It's roundabout: max_rc is being forwarded through this routing to 'copy rexx and run', which changes behavior based on max_rc: it will stop processing commands when it hits a failure.

Old behavior was bad, because it would run command #3 even after command #2 failed.  @ddimatos and I worked through this.  So there are 2 behavioral changes:

1 - If an 'error' hits, processing stops, and the output text contains all reactions including the line that stopped processing.
2 - 'error' has been defined as return code is > max_rc (which is 0 if not provided.)